applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -40,6 +40,16 @@ static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_AUDIO_UNICAST_C
 						       NET_BUF_POOL_PTR_ITERATE, (,)) };
 /* clang-format on */
 
+/* Presentation delay search type */
+enum le_audio_pres_search {
```

Comment:
This enum should be static.

Reply:
Why?
It is not assigning any storage so how can I define it static?

Reply:
This has been removed to make the code consistent with how we configure the system now.